controls/radwizard/getting-started.md

```diff
@@ -19,37 +19,56 @@ This tutorial will walk you through the creation of a sample application that co
 >
 >You can find more info {% if site.site_name == 'Silverlight' %}[here]({%slug installation-installing-controls-dependencies%}){% endif %}{% if site.site_name == 'WPF' %}[here]({%slug installation-installing-controls-dependencies-wpf%}){% endif %}  .
 
-For the purpose of this example, you will need to open Visual Studio and then create an empty Application project.
-
 ## Adding RadWizard to the Project
-You can add __RadWizard__ manually by writing the XAML code in __Example 1__. You can also add the control by dragging it from the Toolbox. 
+You can add __RadWizard__ in XAML or in code. You can also add the control by dragging it from the [Toolbox]({%slug installation-adding-to-vs-2017-toolbox%}). 
+
+#### __[C#] Example 1: Adding a RadWizard in code__
```

Comment:
Can you rotate the order of the snippets? First XAML then C# and then VB. It is not big of a deal, but this is used all across the documentation, so let's be consistent. Also, it bugs me. :D